sdk/eventhub/event-hubs/src/eventHubProducerClient.ts

```diff
@@ -151,12 +161,45 @@ export class EventHubProducerClient {
     return producer.createBatch(options);
   }
 
-  async sendBatch(batch: EventDataBatch, options?: SendOptions): Promise<void>;
-  async sendBatch(batch: EventDataBatch, partitionId: string, options?: SendOptions): Promise<void>;
+  /**
+   * Sends a batch of events to the associated Event Hub.
+   *
+   * @param eventData an instance of `EventDataBatch` (to create a batch use `createBatch`)
+   * @param options The set of options that can be specified to influence the way in which
+   * events are sent to the associated Event Hub.
+   * - `abortSignal`  : A signal the request to cancel the send operation.
+   *
+   * @returns Promise<void>
+   * @throws {AbortError} Thrown if the operation is cancelled via the abortSignal.
+   * @throws {MessagingError} Thrown if an error is encountered while sending a message.
+   * @throws {TypeError} Thrown if a required parameter is missing.
+   * @throws {Error} Thrown if the underlying connection or sender has been closed.
+   * @throws {Error} Thrown if a partitionKey is provided when the producer was created with a partitionId.
+   * @throws {Error} Thrown if batch was created with partitionKey different than the one provided in the options.
+   */
+  async sendBatch(batch: EventDataBatch, options?: SendBatchOptions): Promise<void>;
+  /**
+   * Sends a batch of events to the associated Event Hub to a specific partition
+   *
+   * @param eventData an instance of `EventDataBatch` (to create a batch use `createBatch`)
```

Comment:
```suggestion
   * @param batch An instance of `EventDataBatch` (to create a batch use `createBatch`)
```

Reply:
Fixed with https://github.com/Azure/azure-sdk-for-js/pull/5964/commits/92eb64d98eb395114ec3f7f761db85106344c2f8